hplot/example_s2d_test.go

```diff
@@ -123,3 +123,35 @@ func ExampleS2D_withBand() {
 		log.Fatal(err)
 	}
 }
+
+// ExampleS2D_withHiSteps draws some scatter points
+// with their error bars, using a step-like style
+func ExampleS2D_withStepsKind() {
+	pts := []hbook.Point2D{
+		{X: 1, ErrX: hbook.Range{Min: 0.5, Max: 0.5}, Y: 1, ErrY: hbook.Range{Min: 2, Max: 3}},
+		{X: 2, ErrX: hbook.Range{Min: 0.5, Max: 0.5}, Y: 2, ErrY: hbook.Range{Min: 5, Max: 2}},
+		{X: 3, ErrX: hbook.Range{Min: 0.5, Max: 0.5}, Y: 3, ErrY: hbook.Range{Min: 2, Max: 2}},
+		{X: 4, ErrX: hbook.Range{Min: 0.5, Max: 0.5}, Y: 4, ErrY: hbook.Range{Min: 1.2, Max: 2}},
+	}
+	s2d := hbook.NewS2D(pts...)
+
+	p := hplot.New()
+	p.Title.Text = "Scatter-2D (with steps)"
+	p.X.Label.Text = "X"
+	p.Y.Label.Text = "Y"
+	p.Add(plotter.NewGrid())
+
+	var stepsKind hplot.StepsKind = 1
```

Comment:
```suggestion
	var stepsKind = hplot.HiSteps
```
(or just pass that value directly to `hplot.WithStepsKind`, below.)